pkg/cmd/flags/filter.go

```diff
@@ -121,31 +121,31 @@ type filterFlag struct {
 	full              string
 	filterName        string
 	operatorAndValues string
-	scopeIdx          int
+	policyIdx         int
```

Comment:
and this will be removed as well (and all following changes to filterFlag)?

Reply:
We need to think since it's part of the infrastructure and not exposed. It should be the part or not of the policy name mapping. We'll see.

Reply:
@yanivagman why should we remove the index?

Reply:
But we are using it for policies. Policies are parsed and generate an []filterFlag

Reply:
So at least we better rename this struct and not call it filterFlag.

Reply:
Sure. This should probably be part of the removal of multiscopes from the CLI